tfhe/src/core_crypto/entities/glwe_ciphertext_list.rs

```diff
@@ -161,19 +189,43 @@ impl<Scalar: UnsignedInteger> GlweCiphertextListOwned<Scalar> {
     /// the individual ciphertexts in the list.
     ///
     /// See [`GlweCiphertextList::from_container`] for usage.
-    pub fn new(
-        fill_with: Scalar,
+    pub fn from_elem(
         glwe_size: GlweSize,
         polynomial_size: PolynomialSize,
         ciphertext_count: GlweCiphertextCount,
         ciphertext_modulus: CiphertextModulus<Scalar>,
+        fill_with: Scalar,
     ) -> Self {
-        Self::from_container(
-            vec![fill_with; glwe_ciphertext_size(glwe_size, polynomial_size) * ciphertext_count.0],
-            glwe_size,
-            polynomial_size,
-            ciphertext_modulus,
-        )
+        let container_size = glwe_ciphertext_size(glwe_size, polynomial_size) * ciphertext_count.0;
+        let container = vec![fill_with; container_size];
+        Self::from_container(container, glwe_size, polynomial_size, ciphertext_modulus)
+    }
+
+    /// Allocate memory and create a new owned [`LweCiphertextList`], where each element
+    /// is provided by the `fill_with` function, invoked for each consecutive index.
+    ///
+    /// # Note
+    ///
+    /// This function allocates a vector of the appropriate size and wraps it in the appropriate
+    /// type. If you want to encrypt data in the list you need to use
+    /// [`crate::core_crypto::algorithms::encrypt_glwe_ciphertext_list`] or a variant working on
+    /// a single ciphertext at a time [`crate::core_crypto::algorithms::encrypt_glwe_ciphertext`] on
+    /// the individual ciphertexts in the list.
+    ///
+    /// See [`GlweCiphertextList::from_container`] for usage.
```

Comment:
I don't see a change in the from_container docstring to demonstrate usage ?

Reply:
I added an usage example now.